packages/scripts/scripts/build.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+const { sync: spawn } = require( 'cross-spawn' );
+const { sync: resolveBin } = require( 'resolve-bin' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	getCliArgs,
+	hasCliArg,
+	hasProjectFile,
+} = require( '../utils' );
+
+const hasWebpackConfig = hasCliArg( '--config' ) ||
+	hasProjectFile( 'webpack.config.js' ) ||
+	hasProjectFile( 'webpack.config.babel.js' );
+
+if ( hasWebpackConfig ) {
+	// Sets environment to production.
+	process.env.NODE_ENV = 'production';
+
+	const { status } = spawn(
+		resolveBin( 'webpack' ),
+		[ ...getCliArgs() ],
```

Comment:
This is better, but I'm still not sue why you need to spread to a *new* array here.  why not just pass through `getCliArgs()`?

Reply:
Haha, I see it now. You are totally right 😅